src/domain/messages/entities/schemas/message.schema.ts

```diff
@@ -40,3 +41,8 @@ export const messageSchema: Schema = {
     'confirmations',
   ],
 };
+
+export const messagePageSchema: Schema = buildPageSchema(
+  'https://safe-client.safe.global/schemas/messages/message-page.json',
```

Comment:
Out of context of this PR, but what do you think about moving the `keyRef` strings to constants? We're using them in multiple places and they're easily mistyped.

Reply:
Thank you @iamacook, after a second thought, I think this constants extraction fits better in a separate PR as you suggested!